solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -162,6 +183,10 @@ public boolean split(ClusterState clusterState, ZkNodeProps message, NamedList<O
     }
 
     RTimerTree t;
+    // 1. verify that there is enough space on disk to create sub-shards
+    log.info(
```

Comment:
Let's move this inside checkDiskSpace since sometimes we don't check the disk space actually.